src/components/shared/Avatar/Avatar.tsx

```diff
@@ -4,12 +4,12 @@ import getIcon from './identicon';
 import Icon from '../Icon';
 import styles from './Avatar.css';
 
-export interface Props {
+export interface AvatarProps {
   /** Seed phrase for blockies fallback (usually an address) */
   seed?: string;
 
-  /** Avatar image URL (can be a base64 encoded string) */
-  avatarURL?: string;
+  /** Avatar image (can be a base64 encoded string) */
+  avatar?: string;
```

Comment:
Removing reference to `avatarURL` since we aren't storing data on IPFS any longer